README.md

```diff
@@ -68,14 +74,16 @@ defmodule MyProject.Repo.Migrations.CreateMessage do
 end
 ```
 
-### Database model
+### Schema module:
```

Comment:
Do not use `:` in headings.

Reply:
You're right. Then `:` in
### Database migration:
needs to be removed instead.

Reply:
Yes please.